### PR TITLE
fix:  rename option of ResponseTime

### PR DESCRIPTION
**Pull request by rokamu623**
## Description
The current option names for ResponseTime diverge from the intuitive image.
The option names have been changed as follows:
- worst -> worst-with-external-latency
- worst-in-input -> worst

image

## Related links
Jira: https://tier4.atlassian.net/browse/RT2-888

## Pre-review checklist for the PR author

- [x] I've confirmed the [contribution guidelines](https://github.com/tier4/caret/blob/main/.github/CONTRIBUTING.md).
- [x] The PR is ready for review.

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [x] The PR has been properly tested.
- [x] The PR has been reviewed.

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] (Optional) The PR has been properly tested with CARET_report verification.
- [ ] There are no open discussions or they are tracked via tickets.
- [x] The PR is ready for merge.

After all checkboxes are checked, anyone who has write access can merge the PR.

### CODE REVIEW

File: src/caret_analyze/record/records_service/response_time.py [hunk 1]
```diff
@@ -450,33 +450,46 @@ def __init__(
         self._histogram = ResponseHistogram(self._records, self._timeseries)
 
     def to_all_records(self) -> RecordsInterface:
-        return self._response_map_all.to_all_records()
+        """
+        Calculate the all records data for response time.
```
Comment (bopeng-saitama) [inline src/caret_analyze/record/records_service/response_time.py:454]:
```suggestion
        Calculate the data of all records for response time.
```
or
```suggestion
        Calculate all the records' data for response time.
```

File: src/caret_analyze/record/records_service/response_time.py [hunk 2]
```diff
@@ -450,33 +450,46 @@ def __init__(
...
+            - {'response_time'}
 
-    def to_stacked_bar(self) -> RecordsInterface:
         """
-        Calculate records for stacked bar.
+        return self._response_map_all.to_all_records()
+
+    def to_worst_case_records(self) -> RecordsInterface:
+        """
+        Calculate the worst case records data for response time.
```
Comment (bopeng-saitama) [inline src/caret_analyze/record/records_service/response_time.py:472]:
```suggestion
        Calculate data of the worst case records for response time.
```

File: src/caret_analyze/record/records_service/response_time.py [hunk 3]
```diff
@@ -450,33 +450,46 @@ def __init__(
...
+
+        The worst case for response time are included message flow latency.
```
Comment (bopeng-saitama) [inline src/caret_analyze/record/records_service/response_time.py:474]: This sentence feels unclear in its expression and is difficult to understand; please consider using a different way to convey this sentence.

File: src/caret_analyze/record/records_service/response_time.py [hunk 4]
```diff
@@ -450,33 +450,46 @@ def __init__(
         self._histogram = ResponseHistogram(self._records, self._timeseries)
 
     def to_all_records(self) -> RecordsInterface:
-        return self._response_map_all.to_all_records()
+        """
+        Calculate the all records data for response time.
 
-    def to_worst_in_input_records(self) -> RecordsInterface:
-        return self._response_map_all.to_worst_in_input_records()
+        The all for response time are included message flow latency.
```
Comment (bopeng-saitama) [inline src/caret_analyze/record/records_service/response_time.py:456]: This sentence feels unclear in its expression and is difficult to understand; please consider using a different way to convey this sentence.

File: src/caret_analyze/record/records_service/response_time.py [hunk 5]
```diff
@@ -450,33 +450,46 @@ def __init__(
...
-        return self._records.to_range_records()
+        return self._response_map_all.to_worst_case_records()
 
-    def to_best_case_stacked_bar(self) -> RecordsInterface:
+    def to_best_case_records(self) -> RecordsInterface:
         """
-        Calculate records for stacked bar.
+        Calculate the best case records data for response time.
+
+        The best case for response time are included message flow latency.
```
Comment (bopeng-saitama) [inline src/caret_analyze/record/records_service/response_time.py:492]: This sentence feels unclear in its expression and is difficult to understand; please consider using a different way to convey this sentence.

File: src/caret_analyze/record/records_service/response_time.py [hunk 6]
```diff
@@ -485,12 +498,29 @@ def to_best_case_stacked_bar(self) -> RecordsInterface:
...
 
         """
-        return self._records.to_range_records('best')
+        return self._timeseries.to_best_case_records()
+
+    def to_worst_with_external_latency_case_records(self) -> RecordsInterface:
+        """
+        Calculate the worst-with-external-latency case records data for response time.
+
+        The worst-with-external-latency case in response time includes message flow latencies
```
Comment (bopeng-saitama) [inline src/caret_analyze/record/records_service/response_time.py:510]: This sentence feels unclear in its expression and is difficult to understand; please consider using a different way to convey this sentence.

**Review by bopeng-saitama — CHANGES_REQUESTED** [review]

**Review by bopeng-saitama — DISMISSED** [review]
LGTM

File: src/caret_analyze/record/records_service/response_time.py (hunk 1 above)
Comment (rokamu623) [reply src/caret_analyze/record/records_service/response_time.py]: Fixed it in [a103900](https://github.com/tier4/CARET_analyze/pull/355/commits/a1039004f34c90c7c7f17675a466049aaf7220d5).

File: src/caret_analyze/record/records_service/response_time.py (hunk 2 above)
Comment (rokamu623) [reply src/caret_analyze/record/records_service/response_time.py]: Fixed it in [a103900](https://github.com/tier4/CARET_analyze/pull/355/commits/a1039004f34c90c7c7f17675a466049aaf7220d5).

File: src/caret_analyze/record/records_service/response_time.py (hunk 3 above)
Comment (rokamu623) [reply src/caret_analyze/record/records_service/response_time.py]: Fixed it in [d9d1ce7](https://github.com/tier4/CARET_analyze/pull/355/commits/d9d1ce7accdc922dfe6e445c6623d3c23746ec5d).

File: src/caret_analyze/record/records_service/response_time.py (hunk 4 above)
Comment (rokamu623) [reply src/caret_analyze/record/records_service/response_time.py]: Fixed it in [d9d1ce7](https://github.com/tier4/CARET_analyze/pull/355/commits/d9d1ce7accdc922dfe6e445c6623d3c23746ec5d).

File: src/caret_analyze/record/records_service/response_time.py (hunk 5 above)
Comment (rokamu623) [reply src/caret_analyze/record/records_service/response_time.py]: Fixed it in [d9d1ce7](https://github.com/tier4/CARET_analyze/pull/355/commits/d9d1ce7accdc922dfe6e445c6623d3c23746ec5d).

File: src/caret_analyze/record/records_service/response_time.py (hunk 6 above)
Comment (rokamu623) [reply src/caret_analyze/record/records_service/response_time.py]: Fixed it in [d9d1ce7](https://github.com/tier4/CARET_analyze/pull/355/commits/d9d1ce7accdc922dfe6e445c6623d3c23746ec5d).

**Review by nabetetsu — APPROVED** [review]
I have confirmed that all target options work as expected.
LGTM :+1:

**Pull request merged**